website/docs/concepts/router.md

```diff
@@ -408,23 +408,29 @@ The graph is produced with the following code, with graphviz.
 To reproduce. Save the code in a file, say `input.dot`,
 And run `$ dot -Tgif input.dot  -o nested-router.gif`
 
-graph {
+digraph {
     node [shape=box style=rounded]
-    Home; News; Contact; "Not Found"; Profile; Friends; Theme; "Settings Not Found";
+    Home; News; Contact; "Not Found"; Profile; Friends; Theme; SettingsNotFound [label="Not Found"];
 
     node [fillcolor=lightblue style="filled, rounded"]
     "Main Router"; "Settings Router";
 
-    "Main Router" -- {Home News Contact "Not Found" "Settings Router"}
-    "Settings Router" -- {Profile Friends Theme "Settings Not Found"}
+    "Main Router" -> {Home News Contact "Not Found" "Settings Router"} [arrowhead=none]
+    "Settings Router" -> {SettingsNotFound Profile Friends Theme } [arrowhead=none]
+    SettingsNotFound -> "Not Found" [constraint=false]
 }
 -->
 
 ![nested router diagram](/img/nested-router.gif)
 
+The nested `SettingsRouter` handles all urls that start with `/settings`. Additionally, it redirects urls that are not
+matched to the main `NotFound` route. So `/settings/gibberish` will redirect to `/404`.
+
```

Comment:
Forgot to add the redirect last time. As a tip in the <Redirect/> section above mentioned, the nested router should have an example use of <Redirect/>